api/fees/fees.go

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) 2025 The VeChainThor developers
+
+// Distributed under the GNU Lesser General Public License v3.0 software license, see the accompanying
+// file LICENSE or <https://www.gnu.org/licenses/lgpl-3.0.html>
+
+package fees
+
+import (
+	"math"
+	"net/http"
+	"strconv"
+
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/gorilla/mux"
+	"github.com/pkg/errors"
+	"github.com/syndtr/goleveldb/leveldb"
+	"github.com/vechain/thor/v2/api/utils"
+	"github.com/vechain/thor/v2/bft"
+	"github.com/vechain/thor/v2/cache"
+	"github.com/vechain/thor/v2/chain"
+	"github.com/vechain/thor/v2/thor"
+)
+
+type Fees struct {
+	data *FeesData
+}
+type FeeCacheEntry struct {
```

Comment:
It might be not clear for my previous comment. I was suggesting move `FeeCacheEntry` and `FeesData` to `data.go`. Reason behind that is when I was trying to find the type define near function `New`, put the constructor and struct definition will make the file structure clear, wdyt?

Reply:
Done